hms-app/src/components/HackathonHeader.tsx

```diff
@@ -9,19 +16,30 @@ type IProps = {
 
 export default function HackathonHeader(props: IProps) {
   const { classes } = styles()
+  const theme = useMantineColorScheme()
   const { hackathonData } = props
 
   return (
-    <div>
-      <Text align={'center'} className={classes.title}>
-        Title: {hackathonData.title}
-      </Text>
-      <Text align={'center'} className={classes.title}>
-        Start date: {new Date(hackathonData.startDate).toLocaleDateString()}
-      </Text>
-      <Text align={'center'} className={classes.title}>
-        End date: {new Date(hackathonData.endDate).toLocaleDateString()}
-      </Text>
-    </div>
+    <Container my={30}>
+      <Center>
+        <Title>{hackathonData.title}</Title>
+      </Center>
+      <Center mt={10}>
+        <Badge
+          size={'lg'}
+          variant={'gradient'}
+          gradient={
+            theme.colorScheme === 'dark'
+              ? { from: 'teal', to: 'blue', deg: 60 }
+              : { from: '#ed6ea0', to: '#ec8c69', deg: 35 }
```

Comment:
colors should probably be defined in the colors file so they can be faster found and changed